components/forms/RadioButton/html.html

```diff
@@ -0,0 +1,3 @@
+<div class="radio">
+  <input class="radio__input" key = "r" name ="radio" type="radio" />
```

Comment:
`key` is not a native HTML attribute and can be removed. Please clean up formatting